tests/io/DirectoryTest.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+from coalib.io.Directory import Directory
+
+
+class DirectoryTest(unittest.TestCase):
+
+    def setUp(self):
+        self.test_dir_path = os.path.join(os.path.dirname(__file__),
+                                          'DirectoryTestDir')
+        self.another_test_dir_path = os.path.join(os.path.dirname(__file__),
+                                                  'DirectoryTestDir/Dir1')
```

Comment:
-->
```python
os.path.join(os.path.dirname(__file__), 'DirectoryTestDir', 'Dir1')
```
The `/` inside the path isn't platform independent ;) Let `os.path.join` do the task ;)